src/annotation.js

```diff
@@ -174,7 +176,8 @@ function calculateElementProperties(chart, options, defaults) {
 		y2,
 		width: x2 - x,
 		height: y2 - y,
-		options: merge(Object.create(null), [defaults, options])
+		options: merge(Object.create(null), [defaults, options]),
+		_obliqueLine: oblique
```

Comment:
I'd contain the logic completely to the `line` element, because it only applies to line, right?

Reply:
I have added some logic over there (instead of in line) because annotation has got already the logic there but having a look better, I can put it in line . Let me try